datadog_checks_base/datadog_checks/checks/win/wmi/__init__.py

```diff
@@ -215,12 +215,20 @@ def _submit_metrics(self, metrics, metric_name_and_type_by_property):
         Resolve metric names and types and submit it.
         """
         for metric in metrics:
-            if metric.name not in metric_name_and_type_by_property:
+            if metric.name not in metric_name_and_type_by_property \
+                and str.lower(metric.name) not in metric_name_and_type_by_property:
                 # Only report the metrics that were specified in the configration
                 # Ignore added properties like 'Timestamp_Sys100NS', `Frequency_Sys100NS`, etc ...
+                self.log.debug('cont {} {}'.format(metric.name, metric_name_and_type_by_property))
```

Comment:
Can we have a more meaningful debug message, and then again, do we need to log a debug message? Aka, does this happen frequently enough that it would be considered spam-y and is it even useful to include?

Reply:
should be removed